plugins/hydra_ax_sweeper/example/banana.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+
+import hydra
+
+log = logging.getLogger(__name__)
+
+
+@hydra.main(config_path="conf/config.yaml")
+def banana(cfg):
+    x = cfg.banana.x
+    y = cfg.banana.y
+    z = (1 - x) ** 2 + 1 * ((y - x ** 2) ** 2)
```

Comment:
I realized you completely removed a and b here.
the default values we had before (1 and 100 irrc) are from the wiki page:

> Usually these parameters are set such that a = 1 and b = 100

Can you keep those defaults in the code? (this function)